### PR TITLE
Added script to save data to Google Sheet and addded email support

### PR DESCRIPTION
**Pull request by chaitanyab2311**

### CODE REVIEW

File: src/resource_instances.py [hunk 1]
```diff
@@ -15,18 +16,25 @@ def get_instances():
             resource_details = {'instance_id': resource['id'], 'name': resource['name'], 'region': resource['region_id'],
                                 'resource_group_id': resource['resource_group_id'], 'type': resource['type'],
                                 'resource_id': resource['resource_id'], 'state': resource['state'],
-                                'created_by': get_owner_details(resource['created_by']), 'created_at': resource['created_at'],
-                                'updated_at': resource['updated_at'], 'deleted_at': resource['deleted_at']}
+                                'created_by': '', 'created_at': resource['created_at'],
```
Comment (mundra-ankur) [inline src/resource_instances.py]: Why did you deleted owner deatils?

File: src/resource_instances.py [hunk 2]
```diff
@@ -15,18 +16,25 @@ def get_instances():
...
+                                'updated_at': resource['updated_at']}
             resource_instances.append(resource_details)
 
+            if(resource['type'] == "container_instance"):
```
Comment (mundra-ankur) [inline src/resource_instances.py:31]: I'm not sure if this is true?
container_instance type belongs to cluster?

File: src/usage_report.py [hunk 3]
```diff
@@ -91,8 +91,8 @@ def get_resource_instance_usage(bill_month):
 
 
 def get_all_instances_cost():
-    all_instances = get_instances()
-    all_instances_usage = get_all_resource_instance_usage()
+    all_instances,clusters = get_instances()
+    all_instances_usage = get_all_resource_instance_usage("2022-05")
```
Comment (mundra-ankur) [inline src/usage_report.py]: Pass the date as a parameter to get_all_instances_cost

File: src/resource_instances.py (hunk 1 above)
Comment (chaitanyab2311) [reply src/resource_instances.py]: Yes that was for a dry run the api was throwing an error after requesting the resource multiple times. I will revert and push it

File: src/resource_instances.py (hunk 2 above)
Comment (JonahSussman) [reply src/resource_instances.py:31]: I concur. I think it's possible to add it to the data spat out in `cluster_infrastructure.py`

**Review by mundra-ankur — APPROVED** [review]

**Pull request merged**